src/load/azext_load/data_plane/load_trigger/custom.py

```
@@ -0,0 +1,233 @@
+        logger.info("Updating trigger schedule completed")
+        return response
+    except Exception as e:
+        logger.error("Error occurred while updating trigger schedule: %s", str(e))
+
+def delete_trigger_schedule(
+    cmd,
+    load_test_resource,
+    trigger_id,
+    resource_group_name=None,
+):
+    logger.info(
+        "Deleting trigger schedule with name"
+    )
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    client.delete_trigger(trigger_id)
+    logger.info("Deleting trigger schedule completed.")
+
+
+def get_trigger_schedule(
+    cmd,
+    load_test_resource,
+    trigger_id,
+    resource_group_name=None,
+):
+    logger.info(
+        "Getting trigger schedule with name"
+    )
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    response = client.get_trigger(trigger_id)
+    logger.debug("Fetched trigger schedule: %s", response)
+    return response
+
+
+def pause_trigger_schedule(
+    cmd,
+    load_test_resource,
+    trigger_id,
+    resource_group_name=None,
+):
+    logger.info(
+        "Pausing trigger schedule with name"
+    )
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    result = None
+    try:
+        result = client.get_trigger(trigger_id=trigger_id)
+    except Exception as e:
+        logger.debug("Error occurred while fetching trigger schedule: %s", str(e))
+        logger.error("Trigger schedule with id: %s not found.", trigger_id)
+        return
+    logger.debug("Existing trigger object: %s", result)
+    if result.state == enums.TriggerState.ACTIVE:
+        result.state = enums.TriggerState.PAUSED
+        response = client.create_or_update_trigger(trigger_id=trigger_id, body=result)
+        logger.debug("Paused trigger schedule: %s", response)
+        return response
+    else:
+        logger.error("Trigger schedule is not active. It is in %s state.", result.state.value)
```

Should we throw a CLI Error as well?